lib/pre-commit/template.rb

```diff
@@ -0,0 +1,83 @@
+require "date"
+require "erb"
+require "fileutils"
+
+module PreCommit
+  class Template
+    TEMPLATE_DIR = File.expand_path("../../../templates/gem", __FILE__)
```

Comment:
I was thinking later we could extend the `new` command, then we could have a way to generate more things like just plugin in existing gem, possibly a plugin template could be called from gem template